sympy/stats/crv_types.py

```diff
@@ -3187,27 +3187,28 @@ def Normal(name, mean, std):
     >>> from sympy.stats import Normal, density, E, std, cdf, skewness, quantile, marginal_distribution
     >>> from sympy import Symbol, simplify, pprint
 
-    >>> mu = Symbol("mu")
-    >>> sigma = Symbol("sigma", positive=True)
+    >>> mean = Symbol("mean")
```

Comment:
I would like to leave these as mu, sigma though, because it is more better to use mathematical names than programming names for symbols, and they print more nicely in IPython environment.

Reply:
Understood. Thanks for the prompt reply.